rosetta-source/src/main/rosetta/base-staticdata-asset-common-type.rosetta

```diff
@@ -86,6 +84,15 @@ type ProductTaxonomy extends Taxonomy: <"Specifies the product taxonomy, which i
     condition TaxonomyValue: <"A taxonomy value and product qualifier are mutually exclusive. Choice is optional as it only applies when source exists.">
         optional choice value, productQualifier
 
+type IndexReferenceInformation: <"A class defining information related to Index">
+    indexName string (0..1)  <"The name of the index expressed as a free format string with an associated scheme.">
+		[metadata scheme]
+	indexId string (0..*) <"An index identifier (e.g. RED pair code).">
+		[metadata scheme]
+
+    condition IndexAttributes: <"A required choice condition for either a floating rate or inflation rate index.">
+		required choice indexName, indexId
```

Comment:
This will cause validation errors if both exist.  Should it be `indexName exists or indexId exists` which would allow both to be set?